styles.css

```diff
@@ -59,25 +59,60 @@ body {
     margin-bottom: 5px;
     border: 1px solid #ddd;
     border-radius: 5px;
+   
   }
 
-  
+/*   
 button {
   padding: 5px 10px;
   border: none;
   border-radius: 4px;
   cursor: pointer;
-}
+} */
 
 button.editbtn {
   background-color: #4caf50;
   color: white;
   float: right;
+  padding: 5px 10px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
 }
 
 button.deletebtn {
   background-color: #f44336;
   color: white;
   float: right;
   margin-right: 10px;
+  padding: 5px 10px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+:root{
+  --primary-color:white;
+  --secondary-color:#4caf50;
+  --tertiary-color:black;
+}
+.darktheme{
+  --primary-color:black;
+  --secondary-color:grey;
+  --tertiary-color:white;
 }
+h2{
+  color:var(--tertiary-color)
+}
+
+#darkbtn{
+  background-color: #fff;
+  color:rgb(86, 66, 164);
+  text-decoration: underline;
+  font-family: Arial, sans-serif;
+  font-weight: bold;
+  /* padding: 10px; */
+  /* margin-bottom: 5px; */
+  /* border: 1px solid #ddd; */
+  /* border-radius: 5px; */
+}
```

Comment:
Again commented code! Don't do it without any specific reason.

Reply:
Got it, i will take care of that.